### DISCUSSION TITLE
编译时会出现一个硬件监控支持，还需要输入 是什么情况？

### DISCUSSION THREAD

**Discussion by small-white-rabbit** [Boos4721/openwrt#105] [Bugs/问题反馈]
* Restart config...
*
*
* Hardware Monitoring support
*
Hardware Monitoring support (HWMON) [M/n/y/?] m
  Hardware Monitoring Chip debugging messages (HWMON_DEBUG_CHIP) [N/y/?] n
  *
  * Native drivers
  *
  Analog Devices AD7314 and compatibles (SENSORS_AD7314) [N/m/?] n
  Analog Devices AD7414 (SENSORS_AD7414) [N/m/?] n
  Analog Devices AD7416, AD7417 and AD7418 (SENSORS_AD7418) [N/m/?] n
  Analog Devices ADM1021 and compatibles (SENSORS_ADM1021) [N/m/?] n
  Analog Devices ADM1025 and compatibles (SENSORS_ADM1025) [N/m/?] n
  Analog Devices ADM1026 and compatibles (SENSORS_ADM1026) [N/m/?] n
  Analog Devices ADM1029 (SENSORS_ADM1029) [N/m/?] n
  Analog Devices ADM1031 and compatibles (SENSORS_ADM1031) [N/m/?] n
  Analog Devices ADM1177 and compatibles (SENSORS_ADM1177) [N/m/?] n
  Analog Devices ADM9240 and compatibles (SENSORS_ADM9240) [N/m/?] n
  Analog Devices ADT7310/ADT7320 (SENSORS_ADT7310) [N/m/?] n
  Analog Devices ADT7410/ADT7420 (SENSORS_ADT7410) [N/m/?] n
  Analog Devices ADT7411 (SENSORS_ADT7411) [N/m/?] n
  Analog Devices ADT7462 (SENSORS_ADT7462) [N/m/?] n
  Analog Devices ADT7470 (SENSORS_ADT7470) [N/m/?] n
  Analog Devices ADT7473, ADT7475, ADT7476 and ADT7490 (SENSORS_ADT7475) [N/m/?] n
  Synaptics AS370 SoC hardware monitoring driver (SENSORS_AS370) [N/m/?] n
  Andigilog aSC7621 (SENSORS_ASC7621) [N/m/?] n
  Analog Devices FAN Control HDL Core driver (SENSORS_AXI_FAN_CONTROL) [N/m/?] n
  ASPEED AST2400/AST2500 PWM and Fan tach driver (SENSORS_ASPEED) [N/m/?] n
  Attansic ATXP1 VID controller (SENSORS_ATXP1) [N/m/?] n
  Dallas Semiconductor DS620 (SENSORS_DS620) [N/m/?] n
  Dallas Semiconductor DS1621 and compatibles (SENSORS_DS1621) [N/m/?] n
  FB-DIMM AMB temperature sensor on Intel 5000 series chipsets (SENSORS_I5K_AMB) [N/m/?] n
  Fintek F71805F/FG, F71806F/FG and F71872F/FG (SENSORS_F71805F) [N/m/?] n
  Fintek F71882FG and compatibles (SENSORS_F71882FG) [N/m/?] n
  Fintek F75375S/SP, F75373 and F75387 (SENSORS_F75375S) [N/m/?] n
  Fujitsu Technology Solutions sensor chip Teutates (SENSORS_FTSTEUTATES) [N/m/?] n
  Genesys Logic GL518SM (SENSORS_GL518SM) [N/m/?] n
  Genesys Logic GL520SM (SENSORS_GL520SM) [N/m/?] n
  GMT G760A (SENSORS_G760A) [N/m/?] n
  GMT G762 and G763 (SENSORS_G762) [N/m/?] n
  GPIO fan (SENSORS_GPIO_FAN) [N/m/?] n
  Honeywell Humidicon HIH-6130 humidity/temperature sensor (SENSORS_HIH6130) [N/m/?] n
  ITE IT87xx and compatibles (SENSORS_IT87) [N/m/?] n
  JEDEC JC42.4 compliant memory module temperature sensors (SENSORS_JC42) [N/m/?] n
  Lattice POWR1220 Power Monitoring (SENSORS_POWR1220) [N/m/?] n
  Lineage Compact Power Line Power Entry Module (SENSORS_LINEAGE) [N/m/?] n
  Linear Technology LTC2945 (SENSORS_LTC2945) [N/m/?] n
  Analog Devices LTC2947 High Precision Power and Energy Monitor over I2C (SENSORS_LTC2947_I2C) [N/m/?] n
  Analog Devices LTC2947 High Precision Power and Energy Monitor over SPI (SENSORS_LTC2947_SPI) [N/m/?] n
  Linear Technology LTC2990 (SENSORS_LTC2990) [N/m/?] n
  Linear Technology LTC4151 (SENSORS_LTC4151) [N/m/?] n
  Linear Technology LTC4215 (SENSORS_LTC4215) [N/m/?] n
  Linear Technology LTC4222 (SENSORS_LTC4222) [N/m/?] n
  Linear Technology LTC4245 (SENSORS_LTC4245) [N/m/?] n
  Linear Technology LTC4260 (SENSORS_LTC4260) [N/m/?] n
  Linear Technology LTC4261 (SENSORS_LTC4261) [N/m/?] n
  Maxim MAX1111 Serial 8-bit ADC chip and compatibles (SENSORS_MAX1111) [N/m/?] n
  Maxim MAX16065 System Manager and compatibles (SENSORS_MAX16065) [N/m/?] n
  Maxim MAX1619 sensor chip (SENSORS_MAX1619) [N/m/?] n
  Maxim MAX1668 and compatibles (SENSORS_MAX1668) [N/m/?] n
  Maxim MAX197 and compatibles (SENSORS_MAX197) [N/m/?] n
  MAX31722 temperature sensor (SENSORS_MAX31722) [N/m/?] n
  MAX31730 temperature sensor (SENSORS_MAX31730) [N/m/?] n
  Maxim MAX6621 sensor chip (SENSORS_MAX6621) [N/m/?] n
  Maxim MAX6639 sensor chip (SENSORS_MAX6639) [N/m/?] n
  Maxim MAX6642 sensor chip (SENSORS_MAX6642) [N/m/?] n
  Maxim MAX6650 sensor chip (SENSORS_MAX6650) [N/m/?] n
  Maxim MAX6697 and compatibles (SENSORS_MAX6697) [N/m/?] n
  Maxim MAX31790 sensor chip (SENSORS_MAX31790) [N/m/?] n
  Microchip MCP3021 and compatibles (SENSORS_MCP3021) [N/m/?] n
  Microchip TC654/TC655 and compatibles (SENSORS_TC654) [N/m/?] n
  Moortec Semiconductor MR75203 PVT Controller (SENSORS_MR75203) [N/m/?] n
  National Semiconductor ADCxxxSxxx (SENSORS_ADCXX) [N/m/?] n
  National Semiconductor LM63 and compatibles (SENSORS_LM63) [N/m/?] n
  National Semiconductor LM70 and compatibles (SENSORS_LM70) [N/m/?] n
  National Semiconductor LM73 (SENSORS_LM73) [N/m/?] n
  National Semiconductor LM75 and compatibles (SENSORS_LM75) [N/m/?] n
  National Semiconductor LM77 (SENSORS_LM77) [N/m/?] n
  National Semiconductor LM78 and compatibles (SENSORS_LM78) [N/m/?] n
  National Semiconductor LM80 and LM96080 (SENSORS_LM80) [N/m/?] n
  National Semiconductor LM83 and compatibles (SENSORS_LM83) [N/m/?] n
  National Semiconductor LM85 and compatibles (SENSORS_LM85) [N/m/?] n
  National Semiconductor LM87 and compatibles (SENSORS_LM87) [N/m/?] n
  National Semiconductor LM90 and compatibles (SENSORS_LM90) [N/m/?] n
  National Semiconductor LM92 and compatibles (SENSORS_LM92) [N/m/?] n
  National Semiconductor LM93 and compatibles (SENSORS_LM93) [N/m/?] n
  National Semiconductor LM95234 and compatibles (SENSORS_LM95234) [N/m/?] n
  National Semiconductor LM95241 and compatibles (SENSORS_LM95241) [N/m/?] n
  National Semiconductor LM95245 and compatibles (SENSORS_LM95245) [N/m/?] n
  National Semiconductor PC87360 family (SENSORS_PC87360) [N/m/?] n
  National Semiconductor PC87427 (SENSORS_PC87427) [N/m/?] n
  NTC thermistor support from Murata (SENSORS_NTC_THERMISTOR) [N/m/?] n
  Nuvoton NCT6683D (SENSORS_NCT6683) [N/m/?] n
  Nuvoton NCT6775F and compatibles (SENSORS_NCT6775) [N/m/?] n
  Nuvoton NCT7802Y (SENSORS_NCT7802) [N/m/?] n
  Nuvoton NCT7904 (SENSORS_NCT7904) [N/m/?] n
  Nuvoton NPCM750 and compatible PWM and Fan controllers (SENSORS_NPCM7XX) [N/m/?] n
  POWER8 OCC through I2C (SENSORS_OCC_P8_I2C) [N/m/?] n
  Philips PCF8591 ADC/DAC (SENSORS_PCF8591) [N/m/?] n
  Sensiron humidity and temperature sensors. SHT15 and compat. (SENSORS_SHT15) [N/m/?] n
  Sensiron humidity and temperature sensors. SHT21 and compat. (SENSORS_SHT21) [N/m/?] n
  Sensiron humidity and temperature sensors. SHT3x and compat. (SENSORS_SHT3x) [N/m/?] n
  Sensiron humidity and temperature sensors. SHTC1 and compat. (SENSORS_SHTC1) [N/m/?] n
  Silicon Integrated Systems Corp. SiS5595 (SENSORS_SIS5595) [N/m/?] n
  SMSC DME1737, SCH311x and compatibles (SENSORS_DME1737) [N/m/?] n
  SMSC EMC1403/23 thermal sensor (SENSORS_EMC1403) [N/m/?] n
  SMSC EMC2103 (SENSORS_EMC2103) [N/m/?] n
  SMSC EMC2305 (SENSORS_EMC2305) [N/m/?] (NEW) 

最后这一行需要手动按下回车或者输入括号内才能继续运行，我记得之前编译没有这个的，不知道怎么取消掉

**Comment by Boos4721** [comment]
**FIXED**
